tests/ext/test_picklepersistence.py

```diff
@@ -137,10 +137,10 @@ def bad_pickle_files():
         "pickletest",
     ]:
         Path(name).write_text("(())")
-    yield True
+    return True
```

Comment:
mh, what's the difference between yeild and return in this context?

Reply:
I think it did that cause there's nothing after the `yield`, hence no point in `yield`ing when we can just `return` it